src/python/pants/core/goals/BUILD

```diff
@@ -3,7 +3,13 @@
 
 python_sources()
 
-python_tests(name="tests", sources=["*_test.py", "!*_integration_test.py"])
+python_tests(
+    name="tests",
+    sources=["*_test.py", "!*_integration_test.py"],
+    overrides={
+        "publish_test.py": {"timeout": 70},
```

Comment:
I hit a timeout locally where it took 60.5 seconds, but killed it at 60.3 seconds.